src/main/java/edu/wisc/my/rssToJson/service/RsstoJsonServiceImpl.java

```diff
@@ -51,6 +54,37 @@ public JSONObject getJsonFromURL(String endpoint) {
         }
         jsonToReturn.put("items", entries);
         jsonToReturn.put("status", "ok");
-        return jsonToReturn;
+        return getJSONCleaned(jsonToReturn);
     }
+
+    private JSONObject getJSONCleaned(JSONObject jsonToClean) {
```

Comment:
This feels like a procedural style of programming. Private method to implement the subroutine. Consider modeling this instead as a helper object. What is this? A `JsonCleaner`? A `StringFilter`? Factoring this out as tight object(s) with tight API(s) might make it easy and fun to unit test this tight it of responsibility.